docs/fiddles/features/navigation-history/renderer.js

```diff
@@ -0,0 +1,84 @@
+const backBtn = document.getElementById('backBtn')
+const forwardBtn = document.getElementById('forwardBtn')
+const backHistoryBtn = document.getElementById('backHistoryBtn')
+const forwardHistoryBtn = document.getElementById('forwardHistoryBtn')
+const urlInput = document.getElementById('urlInput')
+const goBtn = document.getElementById('goBtn')
+const historyPanel = document.getElementById('historyPanel')
+
+async function updateButtons () {
+  const canGoBack = await window.electronAPI.canGoBack()
+  const canGoForward = await window.electronAPI.canGoForward()
+  backBtn.disabled = !canGoBack
+  backHistoryBtn.disabled = !canGoBack
+
+  forwardBtn.disabled = !canGoForward
+  forwardHistoryBtn.disabled = !canGoForward
+}
+
+async function updateURL () {
+  urlInput.value = await window.electronAPI.getCurrentURL()
+}
+
+function transformUrl (url) {
```

Comment:
nit: Since the function before this uses uppercase "URL", I think this function definition should follow the same format to match. Would result in `updateURL` and  `transformURL`